docs/hoon/library/1b.md

```diff
@@ -4,13 +4,21 @@ navhome: /docs/
 
 
 # 1b: tree addressing
+
+Check out the Nock [explanation](docs/nock/explanation/) for more
+information on the tree-addressing system.
+
 ### `++cap`
 
 Tree head
 
-Tests whether an `a` is in the head or tail of a noun. Produces the
-cube `%2` if it is within the head, or the cube `%3` if it is
-within the tail.
+Tests whether the tree address `a` is in the head or the tail of a noun.
+Produces the constant atom `%2` if it is within the head (subtree `+2`), or
+the constant atom `%3` if it is within the tail (subtree `+3`).
+
+In the Urbit binary-tree system, a parent at address `n` will always have its
+children at address `2n` (the head) and address `2n+1` (the tail).
+
```

Comment:
This is unclear.  Is the head `+2`?  Or is the head the child at address `2n`?  These alternative definitions coincide when `n` is `+1`, but not in other cases.

I'm not sure this is the place to explain the address system again.  We're already going to have address explanations in two other places in the docs: in the Nock tutorial and the Hoon tutorial.  You link to one of these above.  That seems sufficient.